examples/common/scripts/vtadmin-up.sh

```diff
@@ -47,6 +47,33 @@ done
 # Check one last time
 [[ $(curl -s "http://localhost:${vtadmin_api_port}/api/clusters") == "${expected_cluster_result}" ]] || fail "vtadmin failed to discover the running example Vitess cluster."
 
+# Download nvm and node
+output() {
+  echo -e "$@"
+}
+
+if [[ -z ${NVM_DIR} ]]; then
+    export NVM_DIR="$HOME/.nvm"
+fi
+
+if [[ -z ${NODE_VERSION} ]]; then
+    export NODE_VERSION="16"
+fi
+
+output "\nInstalling nvm...\n"
+
+if [ -d "$NVM_DIR" ]; then
+  output "\033[1;32mnvm is already installed!\033[0m"
+else
+  curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.39.0/install.sh | bash && output "\033[1;32mnvm is installed!\033[0m" || fail "\033[1;32mnvm failed to install!\033[0m" 
+fi
+
+source "$NVM_DIR/nvm.sh"
+
+output "\nConfiguring Node.js $NODE_VERSION\n"
+nvm install $NODE_VERSION || fail "Could not install nvm $NODE_VERSION."
+nvm use $NODE_VERSION || fail "Could not use nvm $NODE_VERSION."
```

Comment:
We can quote the variables here just to make `shellcheck` happy 🙂 

```
nvm install "$NODE_VERSION" || fail "Could not install nvm $NODE_VERSION."
nvm use "$NODE_VERSION" || fail "Could not use nvm $NODE_VERSION."
```

Reply:
Done! 😄